modules/mono/glue/GodotSharp/GodotSharp/Core/Rect2i.cs

```diff
@@ -151,14 +151,23 @@ public Rect2i Expand(Vector2i to)
         }
 
         /// <summary>
-        /// Returns the area of the <see cref="Rect2"/>.
+        /// Returns the area of the <see cref="Rect2i"/>.
         /// </summary>
         /// <returns>The area.</returns>
         public int GetArea()
         {
             return _size.x * _size.y;
         }
 
+        /// <summary>
+        /// Returns the center of the <see cref="Rect2i"/>.
```

Comment:
```suggestion
        /// Returns the center of the <see cref="Rect2i"/>, which is equal
        /// to <see cref="Position"/> + (<see cref="Size"/> / 2).
        /// If <see cref="Size"/> is an odd number, the returned center
        /// value will be rounded towards <see cref="Position"/>.
```

Be sure to update the other C# methods in AABB and Rect2 to look like this (just the first sentence though).

Reply:
Done.